app/bindings/api/v0/bindings/event.rb

```diff
@@ -6,7 +6,7 @@
 OpenAPI spec version: 0.1.0
 
 Generated by: https://github.com/swagger-api/swagger-codegen.git
-Swagger Codegen version: 2.4.9
+Swagger Codegen version: 2.4.13
```

Comment:
@steveburkett using the generate_model_bindings inside Docker will stop this from flipping back and forth.